### ISSUE TITLE
Wrong reference to ControlzEx

### ISSUE THREAD

**Issue by mpventum** [fluentribbon/Fluent.Ribbon#973]
**After the build of project, the fluent ribbon refear to ControlzEx 4.0 instead of 5.0*

![image](https://user-images.githubusercontent.com/2780284/134918363-011239be-7535-450e-a7f0-719e4647fd09.png)

---
### Environment

- Fluent.Ribbon __v8.0.3__
- Windows __10__
- .NET Framework __4.5.2__

**Comment by batzen** [comment]
Have you tried a clean build?
Did you switch versions of Fluent.Ribbon or ControlzEx recently?

**Comment by mpventum** [comment]
yes, i have cleaned and rebuilt several times.
I have switched by Fluent 7.1 to 8.0.3 through nuget. 
It seams that the Fluent 8 throuth nuget have the wrong reference to ControzEx. 
The problem is obvious when the application was started in release

**Comment by batzen** [comment]
Do you only reference Fluent.Ribbon or also ControlzEx?
When creating a new project and just adding Fluent.Ribbon as a reference it references ControlzEx 4.4 and not 5.0.

**Comment by mpventum** [comment]
it seams that there is a problem with linking on a nuget package

these are the installed libraries
![image](https://user-images.githubusercontent.com/2780284/135051331-ec81bc2d-f1dd-4761-bd8a-e7b8e25b72bb.png)

**Comment by batzen** [comment]
Yeah, that's an issue with how i treated version ranges in Fluent.Ribbon 8.0.
This will be fixed in Fluent.Ribbon 9.0.

To workaround that issue please do not use ControlzEx 5.0 with Fluent.Ribbon 8.0.
You can use ControlzEx 4.4.0 instead.
